packages/react/src/components/ComboBox/ComboBox.js

```diff
@@ -255,8 +255,10 @@ const ComboBox = React.forwardRef((props, ref) => {
           // stop the event from propagating to prevent this. This allows the
           // toggleMenu behavior for the toggleButton to correctly open and
           // close the menu.
-          onMouseUp(event) {
-            event.stopPropagation();
+          onMouseUp() {
```

Comment:
Does the above comment still apply? Seems like we still might be running into this without the propagation (first demo is deploy preview, second is from stable)

https://user-images.githubusercontent.com/3901764/128525519-bfe0438d-3544-4d17-9d82-bb054c8cfe0f.mov

Reply:
ok cool, i didn't really understand the comment 100%, but not its clear whats going on there? So what's the workaround?